### DISCUSSION TITLE
Edits not updating/syncing in the Meeting Guide App

### DISCUSSION THREAD

**Discussion by ACK00** [code4recovery/12-step-meeting-list#573] [Q&A] [answered]
Hello,

I have edited an address for an existing meeting and the meeting has now disappeared from the Meeting Guide app. It is showing up fine in the website plug in. I have updated twice and it has been a few days without syncing to the app. Please advise. Does the region impact this at all? It is right on the county line but I would think that would be google maps issue and not the app. Any help. Thanks for your service.

**Comment by brianw-area24** [comment, marked as answer]
I think you'll need to contact the Meeting Guide support for this. It sounds like they might have to change the boundaries for where they accept meeting listings from your site. The support page is at [https://meetingguide.aa.org/](https://meetingguide.aa.org/) When you contact them, be sure to include the URL of your site, and the name and address of the meeting that's not appearing in the app.

Brian
